libs/blocks/text/text.css

```
@@ -30,47 +24,60 @@
   z-index: -1;
 }
 
-.text-block .full-width {
-  align-items: center;
-  display: flex;
-  flex-direction: column;
-  text-align: center;
-  width: 100%;
+.text-block .foreground {
+  max-width: var(--grid-container-width);
+  margin: 0 auto;
 }
 
-.text-block .full-width .heading-XL,
-.text-block .full-width .heading-L {
-  margin: 0 0 var(--spacing-xs);
+[class*=up] .text-block .foreground {
```

Nit: Seems safer to add the dash just in case `up` might apply to something else in the future
```suggestion
[class*=-up] .text-block .foreground {
```

---

Good call